templates/home/api-fr.html

```diff
@@ -0,0 +1,219 @@
+---
+{
+	"title": "Home page - Meta information",
+	"language": "fr",
+	"altLangPrefix": "en",
+	"altLangPage": "api-en.html",
+	"breadcrumb": [
+		{ "title": "GCWeb home", "link": "../index-fr.html" }
+	],
+	"secondlevel": false,
+	"dateModified": "2022-04-27",
+	"share": "true"
+}
+---
+
+<div lang="en">
+	<p>Cette page n'est pas traduite.</p>
```

Comment:
@polmih Can you put your French paragraph outside the `<div lang="en">` since it is not in English? Thanks!
```suggestion
<p>Cette page n'est pas traduite.</p>
<div lang="en">
```

Reply:
Changed! Thanks